modules/microshift-embed-app-rpms-tutorial.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embedding-apps-tutorial.adoc
+
+:_content-type: CONCEPT
+[id="microshift-embed-app-rpms-tutorial_{context}"]
+= Embed application RPMs tutorial
+
+The following tutorial reviews the {product-title} installation steps and adds a description of the workflow for embedding applications. If you are already familiar with `rpm-ostree` systems such as {op-system-ostree-first} and {product-title}, you can go straight to the procedures.
+
+[id="microshift-installation-workflow-review"]
+== Installation workflow review
+The steps used to embed {product-title} into a {op-system-ostree} image can help you understand the steps needed to embed an application. Embedding applications requires a similar workflow.
+
+//larger concept image here
+
+1. To review: To embed {product-title} on {op-system-ostree}, you added the {product-title} repositories to Image Builder. Then, you created a blueprint that declared all the RPMs, container images, files and customizations you needed, including the addition of {product-title}.
```

Comment:
If you are using numbers, I think that leading with then and next is redundant. Also, you can use the same format as with procedures, starting the list with a `.`. That way the steps are autonumbered.